python/understack-workflows/understack_workflows/discover.py

```diff
@@ -29,11 +29,16 @@ def discover_chassis_info(bmc: Bmc) -> ChassisInfo:
 
     attempts_remaining = LLDP_DISCOVERY_ATTEMPTS
     while len(device_info.neighbors) < MIN_REQUIRED_NEIGHBOR_COUNT:
+        lldp_table = {
+            interface.name: f"{interface.remote_switch_mac_address}/{interface.remote_switch_port_name}"
+            for interface in device_info.interfaces
+        }
         logger.info(
-            "%s does not have enough LLDP neighbors (saw %d), need at least %d. ",
+            "%s does not have enough LLDP neighbors, need %d or more, got %s",
```

Comment:
the `len(device_info.neighbors),` is not needed there anymore as the second `%d` was removed